symplyphysics/laws/thermodynamics/infinitesimal_work_in_quasistatic_process.py

```diff
@@ -0,0 +1,46 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## For a process in a closed system, occuring slowly enough for it to be quasi-static (i.e. the system
+## remains in internal physical, but not necessarily chemical, thermodynamic equilibrium), the infinitesimal
+## increment of work done _by_ the system is related to the pressure inside the system and the infinitesimal
+## increment of volume of the system.
+
+# Law: delta W = P * dV
+## delta W - inexact differential of an infinitesimal increment of work done _by_ the system
```

Comment:
`infinitesimal increment` is differential itself. Do you mean second derivative? Or it is a dublicate wording?

Reply:
Infinitesimal describes the kind of increment, because the increment itself can be finite.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/d5111ee0-fb4c-473c-ab05-d25761a92e60)
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/407bdcc7-e37c-4dab-a0f4-f3e2d14f2199)
I take the wording from English-spoken sources

Reply:
Yes but you also mention `inexact differential` which assumes infinitesimal increment. So it is a dublicate wording IMO.

Reply:
I'll move it to a note below